src/components/Notification/Notification.module.scss

```diff
@@ -0,0 +1,291 @@
+$notification-margin-edge: 24px;
+$notification-margin-bottom: 16px;
+$notification-width: 384px;
+$notification-padding-vertical: 16px;
+$notification-padding-horizontal: 24px;
+$notification-padding: $notification-padding-vertical $notification-padding-horizontal;
+$font-size-base: 14px;
+$font-size-lg: $font-size-base + 2px;
+$font-size-sm: $font-size-base - 2px;
+$border-radius-base: 2px;
+$heading-color: fade(#000, 85%);
+
+:global {
```

Comment:
you should delete global
use scss file not a module.scss instead

Reply:
done